dm/config/task.go

```diff
@@ -393,6 +393,15 @@ func (c *TaskConfig) Decode(data string) error {
 	return c.adjust()
 }
 
+// RawDecode loads config from file data.
+func (c *TaskConfig) RawDecode(data string) error {
+	err := yaml.UnmarshalStrict([]byte(data), c)
```

Comment:
return terror.ErrConfigYamlTransform.Delegate(err, "decode task config failed"), no need to check nil